src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -1881,17 +1881,32 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
     if (HWIntrinsicInfo::IsEmbeddedMaskedOperation(intrinsicId))
     {
         LIR::Use use;
-        bool     foundUse = BlockRange().TryGetUse(node, &use);
         JITDUMP("lowering EmbeddedMasked HWIntrinisic (before):\n");
         DISPTREERANGE(BlockRange(), node);
         JITDUMP("\n");
 
+        // Use lastOp to verify if it's a ConditionlSelectNode. For AddSequentialAcross intrinsic, we can absorb the
```

Comment:
can you remove the reference for `AddSequentialAcross`?

Reply:
Done.